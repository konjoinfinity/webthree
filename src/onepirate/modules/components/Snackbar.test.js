import React from "react"
import { render, screen, fireEvent } from '@testing-library/react'
import "@testing-library/jest-dom/extend-expect"
import Snackbar from "./Snackbar"

  it("should render as default snackbar", () => {
    const { container } = render(<Snackbar />)
    expect(container.firstChild).toMatchSnapshot()
  })
