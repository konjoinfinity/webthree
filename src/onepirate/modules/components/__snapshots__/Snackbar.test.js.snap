// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as default snackbar 1`] = `null`;
